spec/support/shared_examples_for_search.rb

```diff
@@ -1,15 +1,15 @@
 # frozen_string_literal: true
 
 RSpec.shared_examples 'renders search validation errors' do
-  it { expect(response).to render_template('searches/new') }
-  it { expect(response).to render_template('searches/_form') }
-  it { expect(response).not_to render_template('searches/_results') }
+  it { expect(response.body).to include('searches/new') }
+  it { expect(response.body).to include('searches/_form') }
+  it { expect(response.body).not_to include('searches/_results') }
```

Comment:
Why are the body supposed to include those template file names? 🤔 

I think you should include some element from
searches/new
searches/_form
and
I think you can remove the line 6 (not to include)